spec/lib/models/task-definition-spec.js

```diff
@@ -105,6 +105,20 @@ describe('Models.Task', function () {
                 expect(this.subject.json).to.equal(true);
             });
         });
+
+        describe('schemaRef', function () {
+            before(function () {
+                this.subject = this.attributes.schemaRef;
+            });
+
+            it('should be optional', function () {
+                expect(!this.subject.required).to.equal(true);
```

Comment:
I want to test this is not a truthy value, not the actual `true`. The `equal` will not work well as my expectation, I finally found the `ok` is what I want:
```javascript
expect('everything').to.be.ok;
expect(1).to.be.ok;
expect(false).to.not.be.ok;
expect(undefined).to.not.be.ok;
expect(null).to.not.be.ok;
```